hydrogen-vm-render-script.js

```diff
@@ -19,18 +19,111 @@ const {
   setupLightboxNavigation,
   ObservableArray,
   TimelineViewModel,
-  TimelineView
+  TimelineView,
+  createMemoryStorage,
+  MockClock,
+  Client,
+  ConsoleLogger,
+  LoadStatus,
+  SyncStatus,
 } = require('hydrogen-view-sdk');
 
 const events = require('./sample-test-messages.json').chunk;
 
-
 // eslint-disable-next-line max-statements
-function mountHydrogen() {
+async function mountHydrogen() {
+  let hasSynced = false;
+  const platform = {
```

Comment:
`Platform` needs to become an interface so that we can make sure our implementation here is up to spec and more future-proof.